src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -2247,16 +2231,16 @@ private IEnumerable<TypeInferenceInfo> InferTypeInWhileStatement(WhileStatementS
                 return CreateResult(SpecialType.System_Boolean);
             }
 
-            private IEnumerable<TypeInferenceInfo> GetCollectionElementType(INamedTypeSymbol type, int parameterIndex)
+            private IEnumerable<TypeInferenceInfo> GetCollectionElementType(INamedTypeSymbol type)
             {
                 if (type != null)
                 {
-                    var parameters = type.GetAllTypeArguments();
```

Comment:
GetAllTypeArgments finds all the type arguments including for parent types -- but since all callers passed 0 for parameterIndex it was moot.

Reply:
lol